package.json

```diff
@@ -94,6 +94,11 @@
       "type": "boolean",
       "default": false,
       "description": "When opening a file, always focus an already-existing view of the file even if it's in a another pane."
+    },
+    "alwaysRecursiveCollapse": {
```

Comment:
I think this reads better as `alwaysCollapseRecursively`. /cc @atom/feedback

Reply:
Agreed. Updated.